config/routes/api/v2.rb

```diff
@@ -223,6 +223,7 @@ class ActionDispatch::Routing::Mapper
             get :releases
             put :refresh_subscriptions
             put :content_override
+            get :content_override, :action => :product_content
```

Comment:
Why not call the action `list_content_overrides` or rename the route to `get :product_content`?